src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        projects = {
+            d['id']: d
```

Comment:
So, my thinking here:

- The most obvious thing is that it's just unnecessary — in practice, this is really just comparing `f(x) == f(x)`, so it's simpler just to compare `x == x` and skip the intermediate step.
- The other more subtle issue is that there's no guarantees other that the conversion being done to get `d['id']` actually is `six.text_type`, or that it won't change in the future. Basically, what I was referring to as `f(x) == f(x)` is actually `f(x) == g(x)` and they just incidentally happen to have the same result at the moment. If this were to change (which is not likely but also not impossible), this would start returning results that didn't actual match up… which would then not be caught in the `attrs[item]` assignment below since it's using `projects,users.get`.

Also, writing this, I thought of two things:

- Is `project` really necessary to be added onto this serializer? For example, we don't have it on `GroupHash`. The only ways to fetch a tombstone are already scoped by project, so is it a reasonable expectation that you can already get (or know how to get) the project details?
- If it is really necessary for this serializer, it should be attached with `projects[item.project_id]` instead of `projects.get(item.project_id)`, since the project should always exist. (This isn't true for the actor, so the `users` bit is fine as-is.)